seahorn/jobs/string_new_from_string/sea.yaml

```diff
@@ -1,2 +1,3 @@
 verify_options:
   horn-bv2-word-size: 1
+  horn-bmc-tactic: 'smtfd' 
```

Comment:
comment that this is needed for sentinel. I'm collecting benchmarks where smtfd performs well and that would help remind of that.